src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1146,7 +1146,7 @@ void goto_instrument_parse_optionst::instrument_goto_program()
     goto_model.goto_functions.update();
   }
 
-  if(cmdline.isset("synthesize-loop-invariants"))
+  if(cmdline.isset("synthesize-loop-invariants-enumerative"))
```

Comment:
It seems this commit has more than just "documentation?" I'd recommend to factor out the actual code changes into a commit of their own, even if that makes it just a 3-or-so lines commit.

Reply:
+1 what @tautschnig says.